adminapi/filters.py

```diff
@@ -233,12 +233,19 @@ def from_obj(cls, obj):
         raise FilterValueError('Invalid object for Startswith')
 
 
-class Overlap(BaseFilter):
+class Overlaps(BaseFilter):
```

Comment:
No imports to change?

Reply:
Possible.  It was again invented by me, exposed to the client side, but not documented.  I find it unlikely for people to rely on it.